### PR TITLE
Makes pools reduce rad emit strength

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

Makes rads less harsh in pools

## Why It's Good For The Game

what if we kissed in the fuel rod storage pool

## Changelog
:cl:
tweak: pools reduce rads emitted now
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
pool's closed due to rads

**Comment by amidoingitright** [comment]
Make a new commit because Travis is spazzing and not finishing his check

**Comment by jupyterkat** [comment]
the check is done

**Comment by amidoingitright** [comment]
there is another travis that hasn't done his check on my side

File: austation/code/_HELPERS/radiation.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/radiation_pulse(atom/source, intensity, range_modifier, log=FALSE, can_contaminate=TRUE)
```
Comment (PowerfulBacon) [inline austation/code/_HELPERS/radiation.dm:1]: Is this meant to start with proc?

Comment (MCterra10) [reply austation/code/_HELPERS/radiation.dm:1]: no, it's overriding a global proc.

**Pull request merged**